sys/src/cmd/exportfs/exportfs.c

```diff
@@ -184,7 +184,6 @@ main(int argc, char **argv)
 	default:
 		usage();
 	}ARGEND
-	USED(argc, argv);
```

Comment:
USED(argc); USED(argv);

Reply:
Seemed good enough for clang - I guess gcc has to be fussy :)